### PR TITLE
Add LinkedIn auth provider

### PR DESCRIPTION
**Pull request by abolkog**
#### Description of what you did:
Added **LinkedIn** auth provider to support login via LinkedIn

### CODE REVIEW

**Comment by lauriejim** [comment]
Hello @abolkog !

Thank you for this integration.

Can you please add the documentation related to the settings of this provider.
Here is the page where you have to add the documentation.
To do so, you will have to update the `docs/v3.x/plugins/users-permissions.md` file.
For the format, please, follow the same as the other provider.

Thank you.

**Comment by abolkog** [comment]
Hey @lauriejim I have added the documentation.

**Comment by abolkog** [comment]
hey @lauriejim any chance we can get this PR reviewed 😅

**Comment by derrickmehaffy** [comment]
Hi @abolkog let me poke @petersg83

**Review by petersg83 — CHANGES_REQUESTED** [review]
Hi ! Thank you for your contribution, I just tried it and it works well :)
I've noticed a few typos, can you fix them please? Then we're good to merge!
Also if you can rebase your branch on master so there are no conflicts it would be perfect. Thanks!

File: docs/v3.x/plugins/users-permissions.md [hunk 1]
```diff
@@ -501,6 +501,40 @@ The use of `ngrok` is not needed.
...
+The use of `ngrok` is not needed.
+
+#### LinkedIn configuration
+
+- Visit the Apps list page <br> [https://www.linkedin.com/developers/apps](https://www.linkedin.com/developers/apps)
+- Click on **Create app** button
+- Fill the information:
+  - **App name**: Strapi auth
+  - **LinkedIn Page**: Enter a LinkedIn page name to associate with the app or click **Create a new LinkedInPage** to create a new one
+  - **App Logo**: Upload a square image that at least 100x100 pixel.
```
[the remark this answers is not in the document]
Comment (abolkog) [reply docs/v3.x/plugins/users-permissions.md]: Fixed

File: docs/v3.x/plugins/users-permissions.md [hunk 2]
```diff
@@ -501,6 +501,40 @@ The use of `ngrok` is not needed.
...
+LinkedIn accepts the `localhost` urls. <br>
+The use of `ngrok` is not needed.
+
+#### LinkedIn configuration
+
+- Visit the Apps list page <br> [https://www.linkedin.com/developers/apps](https://www.linkedin.com/developers/apps)
+- Click on **Create app** button
+- Fill the information:
+  - **App name**: Strapi auth
+  - **LinkedIn Page**: Enter a LinkedIn page name to associate with the app or click **Create a new LinkedInPage** to create a new one
```
[the remark this answers is not in the document]
Comment (abolkog) [reply docs/v3.x/plugins/users-permissions.md]: Fixed

**Comment by abolkog** [comment]
Thanks @petersg83  for the review. I have fixed the typos and rebased to master.

**Comment by petersg83** [comment]
Hi ! Thanks :) It seems you have un unexpected file in your PR : `packages/strapi-plugin-users-permissions/admin/src/components/PopUpForm/index.js`
Can you fix that please? :)

**Comment by abolkog** [comment]
@petersg83 removed the additional file

**Review by petersg83 — APPROVED** [review]
LGTM thanks for your contribution ! :D

**Pull request merged**